### ISSUE TITLE
accessing user's profile info without being logged in

### ISSUE THREAD

**Issue by Mostafa-Elmenbawy** [SEGUC17/Foobar#32]
1: Severity: ( high )

2:Reported: by Mostafa

3:Description: The URL used after clicking "My profile" can be used despite not being logged in to view the same credential info of a personnel profile 

4: Steps to reproduce the issue: login, click "My Profile", copy the URL, logout, paste the URL

5: Expected result: an error message

**Comment by Mostafa-Elmenbawy** [comment]
![screenshot 8](https://cloud.githubusercontent.com/assets/25394220/25582313/9c251222-2e9d-11e7-8f27-a0ad82b22c87.png)

**Comment by markarsanious** [comment]
yeah you can view it , it is okay to view the info of an SP but you won't have access to edit it anyway
